dali/ft/filecopy.cpp

```diff
@@ -1757,7 +1757,21 @@ void FileSprayer::derivePartitionExtra()
 void FileSprayer::displayPartition()
 {
     ForEachItemIn(idx, partition)
+    {
         partition.item(idx).display();
+
+#ifdef _DEBUG
+        if ((partition.item(idx).whichInput >= 0) && (partition.item(idx).whichInput < sources.ordinality()) )
+            LOG(MCdebugInfoDetail, unknownJob,
+                     "   Header size: %d, XML header size: %" I64F "d, XML footer size: %" I64F "d",
```

Comment:
trivial: all are unsigned so should be %u.

Incidentally why is headerSize an unsigned but xmlHeaderLength a offset_t - should be consistent I'd have thought?

Reply:
>trivial: all are unsigned so should be %u.

@AttilaVamos - looks like this one hasn't been addressed.

Reply:
It has been changed in my last commit/push.